File: 백준/Bronze/10871. X보다 작은 수/X보다 작은 수.cc
#include <stdio.h>

int main(void) {
	int n, a, x, i;
	scanf("%d %d", &a, &x);
	int arr[a];
	for(i=0; i<a; i++) {
		scanf("%d", &arr[i]);
	}
	for(i=0; i<a; i++) {
		if(arr[i]<x)
			printf("%d ", arr[i]);
	}
}

File: 백준/Bronze/10871. X보다 작은 수/X보다 작은 수.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner num = new Scanner(System.in);

        int n = num.nextInt();
        int[] arr = new int[n];
        int x = num.nextInt();

        for(int i=0; i<n; i++) {
            arr[i] = num.nextInt();
        }
        for(int i=0; i<n; i++) {
            if(arr[i]<x)
                System.out.print(arr[i] + " ");
        }
    }
}